addons/web/static/tests/views/graph_view_tests.js

```diff
@@ -1846,7 +1846,11 @@ QUnit.module("Views", (hooks) => {
                     <field name="bar"/>
                 </graph>
             `,
-            views: [[false, "search"]],
+            searchViewArch: `
```

Comment:
is it possible to remove the search view arch in serverData?

Reply:
It can be removed, but for the moment it's used on other tests (it's not a simple remove).